terraform/aws/components/gp2gp/locals_env_variables.tf

```diff
@@ -23,6 +23,18 @@ locals {
     {
       name  = "GP2GP_AMQP_BROKERS"
       value = replace(data.aws_mq_broker.nhais_mq_broker.instances[0].endpoints[1], "amqp+ssl", "amqps") # https://www.terraform.io/docs/providers/aws/r/mq_broker.html#attributes-reference
-    }
+    },
+    {
+      name  = "GP2GP_MHS_INBOUND_QUEUE"
+      value = join("-", list(var.environment, "mhs", "inbound"))
+    },
+    {
+      name  = "MHS_INBOUND_QUEUE_NAME"
```

Comment:
This variable / value will need to go into the MHS component. If @LukaszKwasniewski has not yet finished this the AC could get pushed onto his story instead since it is a small change.

Reply:
Removed